README.md

```diff
@@ -701,6 +714,13 @@ const sum = numbers.reduce(
 console.log(sum) //21
 ```
 
+In ES6
```

Comment:
Same

Reply:
I have made the necessary changes as mentioned above